UnityProject/Assets/Scripts/HealthV2/Living/Damage/Trauma/CreatureTraumaManager.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HealthV2
+{
+	public class CreatureTraumaManager : MonoBehaviour
```

Comment:
how should we console this with the changes I'm doing with the factory approach?

Reply:
I don't know, CreatureTraumaManager usually sits on the mob prefab next to livinghealthmasterbase and only serves an API for healing stuff right now.